docs/products/cassandra/howto/manage-cross-cluster-replication.rst

```diff
@@ -0,0 +1,190 @@
+
+   .. code-block:: bash
+
+      CREATE KEYSPACE keyspace_name WITH replication =    /
+      {                                                   /
+        'class': 'NetworkTopologyStrategy',               /
+        'datacenter_1_name': 'number_of_replicas',        /
+        'datacaenter_2_name': 'number_of_replicas'        /
+      }                                                   /
+      AND durable_writes = true;
+
+   .. code-block:: bash
+      :caption: Example
+
+      CREATE KEYSPACE default WITH replication =           /
+      {                                                    /
+        'class': 'NetworkTopologyStrategy',                /
+        'dc_1': '3',                                       /
+        'dc_2': '3'                                        /
+      }                                                    /
+      AND durable_writes = true;
+
+.. topic:: Result
+
+    You've set up the replication factor for your keyspace. Now all data within this keyspace gets replicated to the datacenters according to the specified factor.
+
+.. seealso::
+
+    For more details on the replication factor for Apache Cassandra, see `NetworkTopologyStrategy <https://cassandra.apache.org/doc/4.1/cassandra/cql/ddl.html#networktopologystrategy>`_ in the Apache Cassandra documentation.
+
+Set up the consistency level
+----------------------------
+
+For Apache Cassandra, you can set up the ``CONSISTENCY`` parameter, which regulates when the client can concern an operation as successfully completed. The ``CONSISTENCY`` parameter defines how many nodes need to confirm the operation as finalized before the client can acknowledge the operation as successfully completed.
+
+.. note::
+    
+    You can configure the consistency level in the shell or in a client library. While using the ``cqlsh`` CLI client is convenient for setting up keyspaces or testing, using a client software is recommended for operations in the production environment and bulk actions, such as data imports, data querying, or data reads/ writes from/ to databases.
+
+In the shell
+''''''''''''
+
+.. note::
+    
+    This instruction uses the ``cqlsh`` Cassandra CLI client to configure the consistency level.
+
+1. :doc:`Connect to your service via cqlsh </docs/products/cassandra/howto/connect-cqlsh-cli>`.
+2. Run ``CONSISTENCY;`` to check your current setting for the consistency level.
+
+.. topic:: Expected output
+
+    The query can return, for example, ``Current consistency level is ONE.``, which means that a conformation of an operation completion on one node is enough for this operation to be considered as successfully completed for your service.
+
+3. To set up the consistency level to a specific value, run the ``CONSISTENCY consistency_level_argument;`` query.
+
+.. topic:: Allowed consistency level arguments
+
+    * ANY
+    * ONE
+    * TWO
+    * THREE
+    * QUORUM
+    * ALL
+    * LOCAL_QUORUM
+    * LOCAL_ONE
+    * SERIAL
```

Comment:
Maybe just linking Cassandra docs would be better? since we'd have to remember to keep this in sync as new versions come along/are phased out?